cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -458,7 +458,7 @@ pQueryCmd =
     pQueryStakeAddressInfo :: Parser QueryCmd
     pQueryStakeAddressInfo =
       QueryStakeAddressInfo
-        <$> pFilterByHexEncodedAddress
+        <$> pFilterByAddress
```

Comment:
This is not right. This parses only Shelley payment addresses (and Byron addresses), but this query command wants only Shelley stake addresses.

You would want a variation on `pFilterByAddress` that uses `parseStakeAddress` instead of `parseAddress`

Reply:
Agh, thanks. My mistake.